Sources/Polymorphic.swift

```diff
@@ -19,16 +20,11 @@ public protocol Polymorphic {
 
 extension Polymorphic {
     public var float: Float? {
-        guard let double = double else {
-            return nil
-        }
-
-        return Float(double)
+        return double.flatMap { Float($0) }
```

Comment:
I think this change makes things less clear. I had to do a double take to understand what was happening here now.